cmd/istioctl/inject.go

```diff
@@ -107,10 +107,20 @@ Example usage:
 )
 
 func init() {
+	defaultHub := os.Getenv(inject.DefaultHubEnvVar)
```

Comment:
Long term the plan is kubernetes admission controller which should do all of this injection server-side. 

Shorter term ConfigMap approach could work. There are corner cases to deal with there as well, e.g. ensure user runs `kube-inject` sufficient time after ConfigMap is created/updates so that the most recent proxy version is picked up.